website/mails/mails.py

```diff
@@ -211,6 +211,14 @@ def get_english_article(word):
     'contributor_added_preprint_node_from_osf',
     subject='You have been added as a contributor to an OSF project.'
 )
+PREPRINT_CONFIRMATION = lambda template, provider: Mail(
```

Comment:
Gut feeling says this should be done differently, or at minimum add a comment explaining it.